### PR TITLE
Implement Spyglass

### PR DESCRIPTION
**Pull request by Prim69**
This implements the Spyglass and the sounds related to it.

### CODE REVIEW

File: server/item/spyglass.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package item
+
+import (
+	"time"
+)
+
+// SpyGlass is an item that zooms in on an area the player is looking at, like a telescope.
+type SpyGlass struct{}
```
Comment (Sandertv) [inline server/item/spyglass.go:8]: SpyGlass is a single world. I'd name it as such.

**Comment by Prim69** [comment]
Fixed :)

File: server/item/spyglass.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+// Spyglass is an item that zooms in on an area the player is looking at, like a telescope.
+type Spyglass struct{}
+
+// MaxCount always returns 1.
+func (Spyglass) MaxCount() int {
+	return 1
+}
+
+// Release ...
+func (Spyglass) Release(_ Releaser, _ time.Duration, _ *UseContext) {}
```
Comment (Sandertv) [inline server/item/spyglass.go:16]: Can remove all `_`s here.

**Comment by Sandertv** [comment]
All tested?

**Comment by Prim69** [comment]
Yep
https://cdn.discordapp.com/attachments/931763129136844820/932396751749546004/2022-01-16_17-10-22_Trim.mp4

**Review by Sandertv — APPROVED** [review]
Perfect, looks good! Thanks for the PR!

**Pull request merged**